admin_manual/installation/changing_the_web_route.rst

```diff
@@ -1,48 +1,23 @@
-======================
-Changing the Web Route
-======================
+==========================
+Changing Your ownCloud URL
+==========================
 
-This admin manual assumes that the owncloud server shall be accessible under the web route
-``/owncloud`` -- this is also where the Linux packages make the server appear. You can change this in your Web server configuration, for example from ``https://example.com/owncloud/`` to ``https://example.com/``.
+This admin manual assumes that the ownCloud server is already accessible under the Web route
+``/owncloud``, which is the default at installation, e.g. ``https://example.com/owncloud``. You can change this in your Web server configuration, for example from ``https://example.com/owncloud/`` to ``https://example.com/``. 
 
-Basic system administrator and Apache configuration knowledge is prerequisite.
-Several configuration files need to be kept in sync when changing the Web route location.
+On Debian/Ubuntu Linux edit these files:
 
-On an Ubuntu-14.04 system the following files are typically involved:
-
-- ``/etc/apache2/conf-enabled/owncloud.conf``
+- ``/etc/apache2/sites-enabled/owncloud.conf``
 - ``/var/www/owncloud/config/config.php``
-- ``/var/www/owncloud/.htaccess``
-
-Example: Moving from /owncloud to /
------------------------------------
-
-    Edit the file /etc/apache2/conf-enabled/owncloud.conf to say::
-
-      Alias / "/var/www/owncloud/"
-
-    Edit /var/www/owncloud/config/config.php to say::
-
-      'overwrite.cli.url' => 'http://localhost/',
-
-    Edit the file /var/www/owncloud/.htaccess to say::
-
-      ...
-      #### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
-      ...
-      <IfModule mod_rewrite.c>
-        RewriteBase /
-      ...
-    
 
-    Optionally also set your document root, though this is generally not needed or recommended.
-    Edit the file ``/etc/apache2/sites-enabled/000-default.conf to say``::
+Edit the ``Alias`` in ``/etc/apache2/sites-enabled/owncloud.conf`` to alias your ownCloud directory to the Web server root::
```

Comment:
In this case the owncloud.conf shipped with the packages is a config file rather then a virtual host:

https://github.com/owncloud/administration/blob/0fdf56c1725b1084237c14db5c3132b881daf1bd/jenkins/obs_integration/templates/owncloud-files/9.0/apache_conf_default

which gets installed by the packages to conf-available:

https://github.com/owncloud/administration/blob/0fdf56c1725b1084237c14db5c3132b881daf1bd/jenkins/obs_integration/templates/owncloud/9.2/debian.__PACKNAME__-deps-php5.postinst#L11

The whole document here was added because tons of users where struggling as the packages have automatically enabled oC to be available at http://example.com/owncloud.

From my PoV the previous path should stay like they where in the past.